PROTO_tests/tests/dataframe_test.py

```diff
@@ -182,7 +191,9 @@ def test_dataframe_creation(self, size):
                 "uint": ak.array(pddf["uint"]),
                 "bigint": ak.arange(2**200, 2**200 + size),
                 "bool": ak.array(pddf["bool"]),
-                "segarray": ak.SegArray.from_multi_array([ak.array(x) for x in pddf["segarray"]]),
+                "segarray": ak.SegArray.from_multi_array(
```

Comment:
Can you check your `black` settings?  It seems like this is reformatting way more than necessary.